repo/packages/Q/quobyte/0/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "quobyte",
+  "version": "1.2",
+  "tags": [
+    "mesosphere",
+    "framework",
+    "quobyte",
+    "storage",
+    "filesystem",
+    "scalability",
+    "mesos",
+    "performance",
+    "bigdata"
```

Comment:
tags should NOT include:  mesos, mesosphere, framework or quobyte.  I question scalability and performance.  Tags are for user search and should help to delineate searches.  the words in question are true for all services.
if there are specific use cases that make sense for the framework feel free to add... like analytics, or ?